content/en/resources.yaml

```diff
@@ -17,10 +17,10 @@ resources:
   - youtube: h8G3LM9uIHk
     title: "Toward Full Adoption of GitOps and Best Practices at RingCentral - Ivan Anisimov & Tamao Nakahara"
     date: 2022-10-25
-  - youtube: 7dOKxIYEtvY
+  - youtube: EXp2xAbII_k
     title: "How to Achieve (Actual) GitOps with Terraform and Flux - Priyanka Ravi, Weaveworks"
     date: 2022-10-25 
-  - url: https://youtube.com/playlist?list=PL9lTuCFNLaD0NVkR17tno4X6BkxsbZZfr
+  - url:https://youtu.be/EXp2xAbII_k 
```

Comment:
Not sure this is correct. It replaces the GitOps Days playlist with RingCentral talk above.

Reply:
Thanks I was editing during the meeting